vpnmanager/vpnservicemanager/vpnprotocol/openvpn/src/main/java/com/kape/openvpn/domain/usecases/StartProcessOutputReader.kt

```diff
@@ -30,7 +30,7 @@ import kotlin.coroutines.CoroutineContext
 
 internal class StartProcessOutputReader(
     private val cache: ICache,
-    private val coroutineContext: ICoroutineContext,
+    private val coroutineContext: com.kape.vpnmanager.api.data.externals.ICoroutineContext,
```

Comment:
We probably don't need the package here.

Reply:
will do 👍